src/main/java/com/cloudant/http/interceptors/TimeoutCustomizationInterceptor.java

```diff
@@ -50,12 +42,11 @@ public TimeoutCustomizationInterceptor() {
      * @param connectTimeout value of the connect timeout
      * @param readTimeout    value of the read timeout
      */
-    public TimeoutCustomizationInterceptor(TimeoutOption connectTimeout, TimeoutOption
+    public TimeoutCustomizationInterceptor(ClientBuilder.TimeoutOption connectTimeout,
+                                           ClientBuilder.TimeoutOption
             readTimeout) {
-        this.connectTimeout = (connectTimeout != null) ? connectTimeout.asIntMillis() :
-                DEFAULT_TIMEOUT.asIntMillis();
-        this.readTimeout = (readTimeout != null) ? readTimeout.asIntMillis() : DEFAULT_TIMEOUT
-                .asIntMillis();
+        this.connectTimeout = connectTimeout.asIntMillis();
```

Comment:
These could NPE now, but without a local default value to use this gets a bit awkward. Maybe set to `-1` and then don't apply a value later if it is `<0` unless you can think of something better, or we just doc that neither supplied `TimeoutOption` should be `null`.